Dragging project layers into processing tools (#7311)
qgis/QGIS-Documentation: docs/user_manual/processing/toolbox.rst

algorithm, the structure is similar for all of them. The parameters found in the
table can be of one of the following types.

.. note::

>  Layers can be dragged directly from the :guilabel:`Layers Panel` into the
>  processing dialog. Does work for :guilabel:`Input layer` and for other
>  layer related inputs.

Reviewer: I'm not sure that `Input layer` (at least as gui label) is meaningful here; it's just the label from the alg used in the PR, it can be anything else (join layer, spoke layer, hub layer, layer to import, ...). I think we should use a generic name for the layer selection tool. Or, picking from the PR (though not 100% sold on)
```suggestion
 Layers can be dragged directly from the :guilabel:`Layers Panel` onto the
 layer selection widget in the algorithm dialog.
```
What do you mean by "layer related inputs"?

Author: With "layer related inputs" i meant everything else than input layers. Also wasn't sure how to name it. Since if we state that it works only for map layer comboboxes (what would be the most correct description), most people (I guess) wont know what a map layer combobox is?

Reviewer: Doesn't "layer selection widget" cover this worry? or "layer selection drop-down widget"?
